src/components/App/App.js

```diff
@@ -26,27 +29,39 @@ class App extends Component {
 
     return (
       <React.Fragment>
-        <PerseidsHeader>
-          Greek Latin Demixer
-          <ul className="navbar-nav ml-auto">
-            <li className="nav-item">
-              <a className="nav-link" href="." target="_blank" rel="noopener noreferrer">New Document</a>
-            </li>
-          </ul>
-        </PerseidsHeader>
-        <div className="text-center">
-          <div className="container">
-            {text ? (
-              <EditXml text={text} />
-            ) : (
-              <InputXml onSubmit={this.handleInputXmlSubmit} />
-            )}
+        <BrowserRouter>
+          <div>
+            <Route path={base}>
+              <PerseidsHeader>
+                <I18n t="title" />
+                <ul className="navbar-nav ml-auto">
+                  <li className="nav-item">
+                    <a className="nav-link" href="." target="_blank" rel="noopener noreferrer">
+                      <I18n t="newDoc" />
+                    </a>
+                  </li>
+                </ul>
+              </PerseidsHeader>
+            </Route>
+            <Route path={base}>
```

Comment:
Instead of repeating `<Route path={base}>` three times, I think we can just replace `<React.Fragment>` on line 31 with `<Route path={base}>` and leave the rest of the code unchanged.